com.microsoft.mrtk.core/Subsystems/Hands/IHandsAggregatorSubsystem.cs

```diff
@@ -66,7 +66,8 @@ public interface IHandsAggregatorSubsystem
 
         /// <summary>
         /// Get a read-only reference to the entire hand.
+        /// Joint poses are returned in an order consistent where each index matches up with the <see cref="Microsoft.MixedReality.OpenXR.HandJoint"/> enum
```

Comment:
Oh actually, also: should this reference the MRTK enum?

Reply:
That would be more clear, though it still does bother me that our enum is a translation layer for ultimately adhereing to the OpenXR enum. Making MRTK the "source of truth" on this kind of matter is what we've been trying to avoid in the first place!

Reply:
Agreed! Definitely open to us discussing taking a more concrete dependency on the MR OpenXR package for pieces like this.

Reply:
Removed it for now, we'll look at how we want MRTK to relate to OpenXR moving in a later discussion